Plots/XY/NCL_xy_4.py

```diff
@@ -0,0 +1,112 @@
+import matplotlib.pyplot as plt
+import matplotlib.path as mpath
+
+import geocat.datafiles as gdf
+from geocat.viz import util as gvutil
+
+###############################################################################
+# Read in data:
+
+# Open a netCDF data file using xarray default engine and load the data into xarrays
+ds = xr.open_dataset(gdf.get('netcdf_files/AtmJan360_xy_4.nc'), decode_times=False)
+
+# Extract a slice of the data
+ds = ds['T']
+t = ds.isel(lev=0).drop('lev')
+t = t.isel(lat=32).drop('lat')
+t = t.isel(lon=29).drop('lon')
+
+###############################################################################
+# Plot with standard markers:
+plt.figure(figsize=(8, 8))
+ax = plt.axes()
+
+plt.scatter(t.time, t.data, color='red')
+
+# Use geocat.viz.util convenience function to set titles and labels
+gvutil.set_titles_and_labels(ax, maintitle="Scatter Plot", xlabel=t['time'].long_name, ylabel=t.long_name)
+
+# Use geocat.viz.util convenience function to add minor and major tick lines
+gvutil.add_major_minor_ticks(ax, x_minor_per_major=5, y_minor_per_major=4, labelsize=14)
+
+gvutil.set_axes_limits_and_ticks(ax, xlim=(131155, 131190),
+                                 ylim=(220.0, 232.0),
+                                 xticklabels=[' ', 131160, ' ', 131170, ' ', 131180, ' ', 131190],
+                                 yticklabels=np.arange(220.0, 233.0, 2.0))
+
+plt.show()
+
+###############################################################################
+# Plot with custom markers:
+plt.figure(figsize=(8, 8))
+ax = plt.axes()
+
+# Divide the data into arbitrary sections so each can be drawn with a different marker
+data1 = t.data[0:8]
+time1 = t.time[0:8]
+
+data2 = t.data[8:16]
+time2 = t.time[8:16]
+
+data3 = t.data[16:24]
+time3 = t.time[16:24]
+
+data4 = t.data[24:]
+time4 = t.time[24:]
+
+# marker='s' creates a square. This is from matplotlib.markers
+# This is not to be confused with the kwarg `s` which sets the marker size
+plt.scatter(time1, data1, color='blue', marker='s', label='matplotlib.markers')
+
+# Use a mathematical symbol for a marker
+plt.scatter(time2, data2, color='green', marker='$\Omega$', s=100, label='mathematical symbols')
+
+# Unicode symbol marker
+plt.scatter(time3, data3, color='black', marker='$\u2608$', s=100, label='unicode symbol')
+
+# Create custom path for marker
+verts = [(-0.5, -0.5), (-0.5, 0.5), (0, 0), (0.5, 0.5), (0.5, -0.5), (0, 0)]
+path = mpath.Path(verts)
+plt.scatter(time4, data4, color='red', marker=path, s=100, label='custom path')
+
+# Add legend
+plt.legend()
+
+# Use geocat.viz.util convenience function to set titles and labels
+gvutil.set_titles_and_labels(ax, maintitle="Make your own marker", xlabel=t['time'].long_name, ylabel=t.long_name)
+
+# Use geocat.viz.util convenience function to add minor and major tick lines
+gvutil.add_major_minor_ticks(ax, x_minor_per_major=5, y_minor_per_major=4, labelsize=14)
+
+gvutil.set_axes_limits_and_ticks(ax, xlim=(131155, 131190),
```

Comment:
Can we get the time info from the file rather than hard-coding it here?

Reply:
The minimum time is 131157 and the maximum time is 131187. I could retrieve that data and the round to the nearest multiple of 5 without much issue.

Do you also want me to generalize the `xticklabels`? Only every other tick label should be drawn so I figured this was the simplest solution.

Reply:
Automatically picking nice, round-number tic labels is not the point of this example. On the other hand the NCL version seems to be doing a reasonable job here. Perhaps some sort of convenience function for geocat-viz is in order, but I'm not sure its a must do. I'm going to approve the PR and leave the choice up to you :-)